docs/serving/using-knative-services.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "Using Knative Services"
+weight: 01
+linkTitle: "Using Knative Services"
+type: "docs"
+---
+
+Knative services are used to deploy an application. Knative Services attempt to model a more logical definition of a Service or Application. To achieve this, Services use the concept of Revisions which are each backed by a deployment with 2 Kubernetes Services.   
+
+For more information about Kubernetes Services, see the [Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/service/).   
+
+Each Knative Service is defined by a Route and a Configuration, which have the same name as the Service, contained in a YAML file.
+
+## Before you begin
+
+To use Knative Services, you will need:
+* A Kubernetes cluster with [Knative installed](https://knative.dev/docs/install/index.html).
```

Comment:
I think it would be beneficial to add a conditional statement here with a pointer to "configuring custom domains" so that its clear that there is additional work to just deploying a service before you can access it externally.